src/collections/_documentation/server/filestore.md

```diff
@@ -30,7 +30,24 @@ filestore.options:
 ```yaml
 filestore.backend: 's3'
 filestore.options:
-  access_key: '...'
-  secret_key: '...'
```

Comment:
I don't quite understand why you removed those lines?

Reply:
Basically because authentication is complicated and there’s a separate discussion point. I didn’t want it to look like that’s required parameter.